controllers/api/index.js

```diff
@@ -150,4 +151,51 @@ router.post('/survey', async (req, res) => {
     }
 });
 
+/**
+ * API: Application Expiry
+ */
+router.post('/applications/expiry', async (req, res) => {
+    try {
+        const expiryApplications = await PendingApplication.findApplicationsByExpiry();
+        const expiredReminderUserIds = [],
+        dayReminderUserIds = [],
+        weekReminderUserIds = [],
+        monthReminderUserIds = [],
+        expiredApplicationIds = [];
+        
+        expiryApplications.forEach((application) => {
```

Comment:
Not sure how I feel about this code, S=switching on `true` is a bit confusing and feels like these conditions are queries. Could we handle this logic in database queries, i.e. query for each condition separately?